spec/rubocop/config_spec.rb

```diff
@@ -156,7 +156,7 @@
       end
     end
 
-    context 'when the configration includes an invalid Enforced.+Style' do
+    context 'when the configuration includes an invalid Enforced.+Style' do
```

Comment:
I don't know why this `.+` was in the description in the first place, but I don't think it should be there. 🙂